com.stremio.App.json

```diff
@@ -0,0 +1,175 @@
+{
+    "app-id": "com.stremio.App",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "version": "24.08",
+            "directory": "lib/ffmpeg",
+            "add-ld-path": "."
+        },
+        "org.freedesktop.Platform.VAAPI.Intel": {
+            "version": "24.08",
+            "directory": "lib/intel-vaapi-driver",
+            "add-ld-path": ".",
+            "download-if": "have-intel-gpu",
+            "enable-if": "have-intel-gpu"
+        }
+    },
+    "command": "stremio",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
it's either `wayland` + `fallback-x11` or just `x11`. Not both.

Reply:
you mean it doesn't run with wayland.

Reply:
in that case remove wayland.

Reply:
the window is able to be created under wayland, chromium needs X11